www/js/diary/infinite_scroll_list.js

```diff
@@ -456,7 +456,7 @@ angular.module('emission.main.diary.infscrolllist',[
     if (!alreadyFiltered) {
         $scope.data.displayTrips = $scope.data.allTrips;
     };
-
+    $scope.data.displayTrips.count = $scope.data.displayTrips.length;
```

Comment:
This was added in https://github.com/JGreenlee/e-mission-phone/commit/129004ee0f542974c0e977c99484e539d9973234 but I don't see the other, related changes - e.g. 

`$scope.data.displayTrips.forEach((cTrip) => {` -> `$scope.data.displayTrips.forEach((cTrip, i, displayTrips) => {`  on line 461.

Double check the merge conflict resolution.

Reply:
Ah, it's because this branch displays the totals differently. It doesn't have the "Displaying X/T trips" text - instead, it puts this up in the filter `<select>` --- e.g. `To Label (9/20)`. These counts reflect the number of the list entries shown out of total total, regardless of whether they are trips, places, untracked, etc.
So we don't actually need to have a `displayTrips.count` on this branch.

Reply:
So you should remove this line?

Reply:
This is another TODO to be handled?

Reply:
Irrelevant in the new branch since the filters were organized into their own component